<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlarmDeviceMonitor</class>
 <widget class="QDialog" name="AlarmDeviceMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alarm Device Monitor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="statusLB">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 51, 51);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>상태 : 경보기 연결 해제됨 (재연결 중)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="stopAllPB">
       <property name="text">
        <string>모든 경보기 동작 중지</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Lamp Control Test</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="2" column="0">
       <widget class="QPushButton" name="orangeBPB">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 153, 0)</string>
        </property>
        <property name="text">
         <string>ORANGE</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="orangeSPB">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 153, 0)</string>
        </property>
        <property name="text">
         <string>OFF</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Blinking</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="greenSPB">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(29, 219, 22)</string>
        </property>
        <property name="text">
         <string>OFF</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Light Off</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="greenBPB">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(29, 219, 22)</string>
        </property>
        <property name="text">
         <string>GREEN</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="redBPB">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 51, 51);</string>
        </property>
        <property name="text">
         <string>RED</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="redSPB">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 51, 51);</string>
        </property>
        <property name="text">
         <string>OFF</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Sound Control Test</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <widget class="QPushButton" name="stopSoundPB">
        <property name="text">
         <string>OFF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sound1PB">
        <property name="text">
         <string>TYPE 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sound2PB">
        <property name="text">
         <string>TYPE 2</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
